README.md

```diff
@@ -1,5 +1,192 @@
 # The LLVM Compiler Infrastructure
 
-This directory and its subdirectories contain source code for LLVM,
-a toolkit for the construction of highly optimized compilers,
-optimizers, and runtime environments.
+This directory and its subdirectories contain source code for LLVM, a toolkit for the construction of highly optimized
+compilers, optimizers, and runtime environments.
+
+## RTC instructions
+
+RTC uses a few llvm tools that require custom fixes, custom checks are are not officially part of the llvm project quite
```

Comment:
`that require custom fixes and checks that are not... `?

double are are already on the line.

Reply:
Nice catch